Makefile

```diff
@@ -1,12 +1,11 @@
-CURL=curl -sSf
+CURL:=curl -sSf
 
-IMAGE_NAME := quay.io/coreos/flannel-cni
-VERSION ?= latest
-TAG ?= 0.1
-CNI_VERSION ?= v0.5.2
+FLANNEL_CNI_ROOT:=$(shell git rev-parse --show-toplevel)
+IMAGE_NAME:=quay.io/coreos/flannel-cni
+VERSION:=$(shell $FLANNEL_CNI_ROOT/scripts/git-version)
```

Comment:
I guess you're not planning to match them up and only want to use it for curl's. Disregard